v2.0/resourcepacks/create_astral_translation/assets/createaddition/lang/zh_cn.json

```
@@ -0,0 +1,214 @@
+  "item.createaddition.iron_rod": "铁棒",
+  "item.createaddition.steel_spool": "钢线轴",
+  "item.createaddition.steel_wire": "钢线",
+  "item.createaddition.steel_rod": "钢棒",
+  "item.createaddition.electrum_spool": "琥珀金线轴",
+  "item.createaddition.electrum_wire": "琥珀金线",
+  "item.createaddition.electrum_rod": "琥珀金棒",
+  "item.createaddition.aluminum_spool": "铝线轴",
+  "item.createaddition.aluminum_wire": "铝线",
+  "item.createaddition.aluminum_rod": "铝棒",
+  "item.createaddition.gold_spool": "金线轴",
+  "item.createaddition.gold_spool.tooltip": "线轴",
+  "item.createaddition.gold_spool.tooltip.summary": "右击两个_连接器_来产生连接它们之间的线缆并返还一个_空线轴_。",
+  "item.createaddition.gold_wire": "金线",
+  "item.createaddition.gold_rod": "金棒",
+  "item.createaddition.brass_rod": "黄铜棒",
+  "item.createaddition.bronze_rod": "青铜棒",
+  "item.createaddition.cake_base": "蛋糕底座",
+  "item.createaddition.cake_base_baked": "烘焙蛋糕底座",
+  "item.createaddition.straw": "吸管",
+  "item.createaddition.seed_oil_bucket": "籽油桶",
+  "fluid.seed_oil": "籽油",
+  "item.createaddition.bioethanol_bucket": "生物乙醇桶",
+  "fluid.bioethanol": "生物乙醇",
+  "item.createaddition.biomass": "生物质",
+  "item.createaddition.biomass_pellet": "生物质颗粒",
+  "item.createaddition.biomass_pellet.tooltip": "BIOMASS_PELLET",
+  "item.createaddition.biomass_pellet.tooltip.summary": "可以在_熔炉_中熔炼_5 个物品_的_固体燃料_。",
+  "item.createaddition.charging_chromatic_compound": "正在充能的异彩化合物",
+  "item.createaddition.overcharged_alloy": "超负荷合金",
+  "item.createaddition.overcharged_alloy.tooltip": "超负荷合金",
+  "item.createaddition.overcharged_alloy.tooltip.summary": "一种由_电力_铸成的异彩材料",
+  "item.createaddition.overcharged_hammer": "超负荷锤",
+  "item.createaddition.overcharged_hammer.tooltip": "超负荷锤",
+  "item.createaddition.overcharged_hammer.tooltip.summary": "这把可靠的_锤子_被注入了_电能_的力量",
+  "item.createaddition.spool": "空线轴",
+  "item.createaddition.spool.nbt": "存储连接地点",
+  "item.createaddition.spool.tooltip": "线轴",
+  "item.createaddition.spool.tooltip.summary": "右击两个_连接器_会移除连接它们之间的线缆并返还一个_线轴_。",
+  "statusbar.createaddition.wire.linked": "节点已连接",
+  "statusbar.createaddition.wire.linked_in": "线缆已连接至输入端",
+  "statusbar.createaddition.wire.linked_out": "线缆已连接至输出端",
+  "statusbar.createaddition.wire.connect": "线缆已连接",
+  "statusbar.createaddition.wire.connect_in": "线缆连接器已选择输入端",
+  "statusbar.createaddition.wire.connect_out": "线缆连接器已选择输出端",
+  "statusbar.createaddition.wire.long": "线缆太长了",
+  "statusbar.createaddition.wire.obstructed": "线缆路径受阻",
+  "statusbar.createaddition.wire.count": "线缆节点已满",
+  "statusbar.createaddition.wire.removed": "线缆已移除",
+  "statusbar.createaddition.wire.exists": "线缆连接已存在",
+  "statusbar.createaddition.wire.no_connection": "这里没有线缆连接",
+  "statusbar.createaddition.wire.invalid": "无效连接",
+  "block.createaddition.electric_motor": "电动马达",
+  "deprecated.block.createaddition.electric_motor.tooltip": "电动马达",
+  "deprecated.block.createaddition.electric_motor.tooltip.summary": "从 _FE_ 产生_应力_并且可调节_速度_。能量消耗基于设置的 _RPM_。",
```

```suggestion
  "deprecated.block.createaddition.electric_motor.tooltip.summary": "从 _FE_ 产生_应力_并且可调节_速度_。能量消耗量和设置的 _RPM_有关。",
```
我不知道这两侧各一个下划线是干啥的

基于RPM怎么读怎么怪

---

你进游戏看看就知道了，下划线是**加粗**，不像markdown是 _斜体_